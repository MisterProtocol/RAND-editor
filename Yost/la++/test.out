Mon May 2 12:00:27 PDT 1988 begin
make test
ccxx -DNEWARGS -V  o/test.o libla.a ../ff3/libff.a
ld: -V: bad flag
	designer C++ Programming System, release 1.2 Sun-Wk.1
	Copyright 1987, Glockenspiel Ltd., Dublin. Ireland
Error executing command "cc -V -o o/test o/test.o ./libla.a ../ff3/libff.a /lib/libcxx.a".
C++ Driver terminating.
*** Error code 4

Stop.
Done
Mon May 2 12:00:31 PDT 1988 end
