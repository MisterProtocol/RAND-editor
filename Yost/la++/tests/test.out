1
2
8
9
10
11
12
13
14
15
16
